.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,4 +2,4 @@ blank_issues_enabled: false
 contact_links:
   - name: Discord community
     url: https://discord.gg/hScy8KWACQ
-    about: Found any bug? Need help? Feel free to contact us via Discord.
+    about: Found any bugs? Need help? Feel free to contact us via Discord.
```

Comment:
I think we should leave that word in singular.

Reply:
Sure, but it's gramatically wrong that way. Could change it to `found a bug?` though :)

Reply:
Agreed! `Found a bug?` it is.
And thank you for this new grammar info. 😀

Reply:
```suggestion
    about: Found a bug? Need help? Feel free to contact us via Discord.
```